src/Install.php

```diff
@@ -298,6 +298,8 @@ protected static function get_schema() {
 			status varchar(255) NOT NULL,
 			is_primary boolean DEFAULT 0 NOT NULL,
 			actioned_text varchar(255) NOT NULL,
+			nonce_action varchar(255) NULL DEFAULT NULL,
```

Comment:
Do we need to perhaps allow `NULL` because not all notes need a nonce?

Reply:
@roykho Yeah that's my intent here. Many notes will never have values given for the nonce fields. Is the SQL wrong?

Reply:
Hmm perhaps I made the wrong conclusion because I am seeing this in my logs `nonce cannot be null...`. But now I am reading it through, it may be related to something else. Here is the full log.

```
WordPress database error Column 'nonce_action' cannot be null for query REPLACE INTO `wp_wc_admin_note_actions` (`note_id`, `name`, `label`, `query`, `status`, `is_primary`, `actioned_text`, `nonce_action`, `nonce_name`) VALUES (51, 'update-db_run', 'Update WooCommerce Database', 'http://gitwooplugins.test/wp-admin/index.php?do_update_woocommerce=true&_nonce_action=wc_db_update&_nonce_name=wc_db_update_nonce', 'unactioned', 1, '', NULL, NULL) made by require_once('wp-admin/admin.php'), set_current_screen, WP_Screen->set_current_screen, do_action('current_screen'), WP_Hook->do_action, WP_Hook->apply_filters, WC_Notes_Run_Db_Update::show_reminder, WC_Notes_Run_Db_Update::update_needed_notice, WC_Data->save, WC_Data_Store->update, Automattic\WooCommerce\Admin\Notes\DataStore->update, Automattic\WooCommerce\Admin\Notes\DataStore->save_actions
[07-Apr-2021 02:46:50 UTC] WordPress database error Column 'nonce_action' cannot be null for query REPLACE INTO `wp_wc_admin_note_actions` (`note_id`, `name`, `label`, `query`, `status`, `is_primary`, `actioned_text`, `nonce_action`, `nonce_name`) VALUES (51, 'update-db_learn-more', 'Learn more about updates', 'https://docs.woocommerce.com/document/how-to-update-woocommerce/', 'unactioned', 0, '', NULL, NULL) made by require_once('wp-admin/admin.php'), set_current_screen, WP_Screen->set_current_screen, do_action('current_screen'), WP_Hook->do_action, WP_Hook->apply_filters, WC_Notes_Run_Db_Update::show_reminder, WC_Notes_Run_Db_Update::update_needed_notice, WC_Data->save, WC_Data_Store->update, Automattic\WooCommerce\Admin\Notes\DataStore->update, Automattic\WooCommerce\Admin\Notes\DataStore->save_actions
```

Reply:
Yeah I think my initial assessment is correct. It is not allowing NULL. Were you able to see this error message?

Reply:
Oh apologies,  I made changes in eaa84ef to start using `NULL`  after you first tested this. I think your database table needs to be migrated to the new structure 😅 

Could you please delete the nonce columns with SQL:

```
ALTER TABLE `wp_wc_admin_note_actions` DROP `nonce_action`;
ALTER TABLE `wp_wc_admin_note_actions` DROP `nonce_name`;

They can then be re-created by triggering a DB migration by de-activating/re-activating the plugin.